CMakeLists.txt

```diff
@@ -6,6 +6,10 @@ set(IsotropicWavelets_LIBRARIES IsotropicWavelets)
 if(NOT ITK_SOURCE_DIR)
   find_package(ITK REQUIRED)
   list(APPEND CMAKE_MODULE_PATH ${ITK_CMAKE_DIR})
+  # Hide clang warnings about undefined-var-template
+  if (CMAKE_CXX_COMPILER_ID MATCHES "Clang") # using regular Clang or AppleClang
+    list(APPEND CMAKE_CXX_FLAGS "-Wno-undefined-var-template")
```

Comment:
A bit unexpected, but `CMAKE_CXX_FLAGS` is a string instead of a list, so 

```
set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-undefined-var-template")
```

Reply:
Thanks @thewtex!